.travis.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+sudo: required
+
+language: cpp
+
+services:
+  - docker
+
+env:
+  matrix:
+    - MESOS_VERSION=1.4.1-2.0.1
+    - MESOS_VERSION=1.3.2-2.0.1
+    - MESOS_VERSION=1.2.3-2.0.1
```

Comment:
Should we drop 1.2 and add 1.5 instead?

Reply:
1.5 will require code adjustments to work. Thus, I left it out for now. The older version I added so we'll see when we break them (and can document that they are no longer supported). This will probably happen once we add 1.5.